packages/vm/test/tester/config.ts

```diff
@@ -129,32 +129,34 @@ export const SKIP_SLOW = [
  * @returns {String} Either an alias of the forkConfig param, or the forkConfig param itself
  */
 export function getRequiredForkConfigAlias(forkConfig: string) {
+  const indexOfPlus = forkConfig.indexOf('+')
+  const remainder = indexOfPlus !== -1 ? forkConfig.substring(indexOfPlus) : ''
   // Chainstart is also called Frontier and is called as such in the tests
   if (String(forkConfig).match(/^chainstart$/i)) {
-    return 'Frontier'
+    return 'Frontier' + remainder
```

Comment:
I'm assuming these names don't change often, if ever, in the test suite, right?

Reply:
I don't recall that ever happening, and if it happens we can update the alias :)